src/login/logind.c

```diff
@@ -1090,6 +1090,8 @@ static int manager_startup(Manager *m) {
         if (r < 0)
                 log_warning_errno(r, "Button enumeration failed: %m");
 
+        manager_load_scheduled_shutdown(m);
```

Comment:
maybe cast to `(void)` explicitly, given you knowingly ignore an error return here.

Reply:
(static analyzers tend to check for stuff like that)

Reply:
I'll make it return void, it's the only user here after all